packages/transition-frontend/src/components/forms/node/TransitNodeEdit.tsx

```diff
@@ -208,6 +207,126 @@ class TransitNodeEdit extends SaveableObjectForm<Node, NodeFormProps, NodeFormSt
         });
     };
 
+    private fillAutocompleteName = async () => {
```

Comment:
Just saying, if you put it in a nice utility function somewhere in a service instead of the component (and this is not really related to a react component's structure), then you can add as many unit tests as you can think of!! The autocomplete list for the code is well unit tested in its service file...

And if you put it in a utility function, it WILL be used again elsewhere at some point...

Reply:
done!